js/main.js

```diff
@@ -202,6 +203,11 @@ Site.Layout = {
 
     if ($('body').hasClass('single-collection')) {
       _this.convertScroll();
+      
+      $('img.single-collection-item').one('load',function() {
```

Comment:
this runs on every image. so that the container width will continue updating as images load.